src/main/webapp/app/exam/participate/exam-participation.component.ts

```diff
@@ -804,9 +804,8 @@ export class ExamParticipationComponent implements OnInit, OnDestroy, ComponentC
                 filter((submissionStateObj) => submissionStateObj != undefined),
                 distinctUntilChanged(),
                 tap((submissionStateObj) => {
-                    const exerciseForSubmission = this.studentExam.exercises?.find(
-                        (programmingExercise) =>
-                            programmingExercise.studentParticipations?.some((exerciseParticipation) => exerciseParticipation.id === submissionStateObj.participationId),
+                    const exerciseForSubmission = this.studentExam.exercises?.find((programmingExercise) =>
+                        programmingExercise.studentParticipations?.some((exerciseParticipation) => exerciseParticipation.id === submissionStateObj.participationId),
```

Comment:
Only a side note as the client style is successful: Can we adapt this in style or in today? Otherwise, we will have constant codacy issues in the future.